phoenix-core/pom.xml

```diff
@@ -493,6 +493,33 @@
 
   </dependencies>
 
+<profiles>
+  <profile>
+    <id>phoenix-hbase-compat-2.3.0</id>
+    <!-- keep enforcer plugin happy -->
+      <activation>
+        <property>
+          <name>hbase.profile</name>
+          <value>2.3</value>
+        </property>
+      </activation>
+    <properties>
+      <dnsjava.version>2.1.7</dnsjava.version>
+    </properties>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.zookeeper</groupId>
+        <artifactId>zookeeper-jute</artifactId>
+      </dependency>
+      <dependency>
+        <groupId>dnsjava</groupId>
+        <artifactId>dnsjava</artifactId>
```

Comment:
Why's this coming in?

YARN was using dnsjava, but I didn't think we had anything directly in hbase or phoenix.

Reply:
No Idea.
Dependency plugin seems to get confused sometimes. (Maybe some unused return value ?)
I've removed the dependency, and added an exception to the dependency plugin config.